letsencrypt/plugins/manual.py

```diff
@@ -159,6 +159,7 @@ def _perform_single(self, achall):
                     # don't care about setting stdout and stderr,
                     # we're in test mode anyway
                     shell=True,
+                    executable="/bin/bash",
```

Comment:
Careful, some systems don't have `bash` out of the box, e.g. FreeBSD. Could we make `executable="/bin/bash"` conditional on Mac OS X presence? Shouldn't it be installed in #883 bootstrap scripts?